test/engine/GroupBySamplingTest.cpp

```diff
@@ -0,0 +1,172 @@
+// Copyright 2025, University of Freiburg,
+// Chair of Algorithms and Data Structures.
+// Author: Benke Hargitai <hargitab@cs.uni-freiburg.de>
+
+#include <gtest/gtest.h>
+
+#include <future>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "../../src/engine/GroupByImpl.h"
+#include "../../src/engine/QueryExecutionContext.h"
+#include "../../src/engine/QueryExecutionTree.h"
+#include "../../src/engine/Result.h"
+#include "../../src/engine/Values.h"
+#include "../../src/engine/idTable/IdTable.h"
+#include "../../src/global/RuntimeParameters.h"
+#include "../../src/parser/GraphPatternOperation.h"
```

Comment:
Use absolut include paths for everything inside `src` (according to the style that all the other files use).